blog/introduccion-typescript.md

```diff
@@ -0,0 +1,261 @@
+# Introduccion a TypeScript
+
+Si bien sabemos que _[todo el software esta roto](https://www.youtube.com/watch?v=0riSpvps4mA)_, podemos utilizar herramientas que nos permitan reducir los errores que podamos cometer a la hora de programar. A lo largo de este post vamos a ver como TypeScript es una herramienta que nos puede ayudar a escribir código "menos roto" en JavaScript.
```

Comment:
"menos roto" en JavaScript. Y cómo [algunos afirman](https://blog.acolyer.org/2017/09/19/to-type-or-not-to-type-quantifying-detectable-bugs-in-javascript/) tranquilamente te pueden ayudar a prevenir alrededor del 15% de tus bugs.